sdk/core/azure-core/inc/azure/core/export.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
```

Comment:
Consider renaming this and elsewhere to be:
`dll_export_windows.hpp`, or at least `dll_export.hpp`